src/lib/dev-environment/dev-environment-core.ts

```diff
@@ -608,13 +613,14 @@ async function updateWordPressImage( slug: string ): Promise<boolean> {
 		if ( currentWordPressTag === 'trunk' ) {
 			return false;
 		}
-	} catch ( error ) {
+	} catch ( err: unknown ) {
+		const error = err as Error & {code: string}
 		// This can throw an exception if the env is build with older vip version
 		if ( 'ENOENT' === error.code ) {
 			message = 'Environment was created before update was supported.\n\n';
 			message += 'To update environment please destroy it and create a new one.';
 		} else {
-			message = `An error prevented reading the configuration of: ${ slug }\n\n ${ error }`;
+			message = `An error prevented reading the configuration of: ${ slug }\n\n ${ error.message }`;
```

Comment:
Here's another runtime change. I think previously it just shows the Error class's name.

Ideally we should also show `error.stack` honestly, but we have debug mode for that purpose.